examples/http-access-logs/http_access_logs.client.out

```diff
@@ -1,3 +1,3 @@
-#Invoke the service.
+# Invoke the service.
 $ curl -k http://localhost:9095/hello
 Successful
```

Comment:
Shall we add a new line at the end of file?

Reply:
Fixed with https://github.com/ballerina-platform/ballerina-lang/pull/12811